tests/foreman/ui/test_activationkey.py

```diff
@@ -1290,11 +1288,45 @@ def test_delete_manifest(self):
 
         @Assert: Deleting a manifest removes it from the Activation key
 
-        @Status: Manual
-
-        @BZ: 1078676
-
         """
+        # Upload manifest
+        manifest_path = manifests.clone()
+        entities.Organization(
```

Comment:
By creating a new organization, we ensure an isolated environment and in the future we don't have to worry about if for some reason start using manifests in the setup.